clustered/server/src/test/java/org/ehcache/clustered/server/state/InvalidationTrackerImplTest.java

```diff
@@ -18,44 +18,97 @@
 
 import org.junit.Test;
 
-import java.util.concurrent.ConcurrentMap;
-
-import static org.hamcrest.Matchers.is;
-import static org.hamcrest.Matchers.nullValue;
-import static org.junit.Assert.*;
+import static org.assertj.core.api.Assertions.assertThat;
 
 public class InvalidationTrackerImplTest {
 
+  private final InvalidationTrackerImpl tracker = new InvalidationTrackerImpl();
```

Comment:
No. That's not the way JUnit works. Each test is ran using a brand new test class instance.